swagger.yaml

```diff
@@ -3997,3 +4004,17 @@ definitions:
     properties:
       config:
         type: string
+
+  static-ip-config:
```

Comment:
Maybe there should be a pattern here? @avishayt ?

Reply:
what do you mean by pattern?

Reply:
for example:
```
pattern: '^(?:(?:(?:[0-9]{1,3}\.){3}[0-9]{1,3}\/(?:(?:[0-9])|(?:[1-2][0-9])|(?:3[0-2])))|(?:(?:[0-9a-fA-F]*:[0-9a-fA-F]*){2,})/(?:(?:[0-9])|(?:[1-9][0-9])|(?:1[0-1][0-9])|(?:12[0-8])))$'
```

Reply:
done